linux_os/guide/system/accounts/accounts-session/accounts_user_interactive_home_directory_defined/ansible/shared.yml

```diff
@@ -0,0 +1,24 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Get all local users from /etc/passwd
+  ansible.builtin.getent:
+    database: passwd
+    split: ':'
+
+- name: Create local_users variable from the getent output
+  ansible.builtin.set_fact:
+    local_users: '{{ ansible_facts.getent_passwd|dict2items }}'
+
+- name: Ensure interactive users have a home directory defined
+  ansible.builtin.user:
+    name: '{{ item.key }}'
+    home: '/home/{{ item.key }}'
```

Comment:
So this assumption may not always be true and if the interactive user has a non-standard home directory this appears to . change that. Maybe we can add when condition this to only run if the user has no home dir.

Reply:
I think it should check the existence of the path defined passwd.

Reply:
Hum, good point @Mab879 . Indeed, without a conditional, the remediation could also change something not expected to be changed. I will include this test. Thanks.